### DISCUSSION TITLE
Feature discussion: Gifs, Arrows

### DISCUSSION THREAD

**Discussion by l3td33r** [rbreu/beeref#51] [Ideas & General Discussion]
Hello, cool software! Thanks for creating it.

My suggestions:
1. Support for autoplaying gifs. This might be demanding for CPU if there are hundreds of gifs on screen, that's why I suggest to autoplay only ~30 gifs that are near the center of the screen, so they're visible when zoomed in.
2. Creating black (or custom color) directional arrows, with user-defined length. Suggested hotkey: Ctrl + Drag Left Mouse Button to create arrow from one place to another. This will allow for mindmap-esque workflow. But I'm not sure whether they should be created (A) only from one object to another (image, text), or (B) from one place to another, no matter if it's empty place on screen or objects. I suppose the latter is easier to do and possibly better?
3. Optional color borders for images, like in PureRef

**Comment by Xiarno** [comment]
This and also my suggestion: https://github.com/rbreu/beeref/discussions/38#discussion-3642442
